UserApi/UserApi/Services/UserAccountService.cs

```diff
@@ -94,7 +86,10 @@ public async Task<User> GetUserByFilter(string filter)
             {
                 var queryResponse = await responseMessage.Content
                     .ReadAsAsync<AzureAdGraphQueryResponse<AzureAdGraphUserResponse>>();
-                if (!queryResponse.Value.Any()) return null;
+                if (!queryResponse.Value.Any())
```

Comment:
value maybe null, use null conditional operator

Reply:
Should never be null (in which case something's wrong). If the request fails it'll be caught further up.